src/components/Slider/Slider.module.css

```diff
@@ -179,3 +179,14 @@
 .slider__marker:not(:first-child):not(:last-child) {
   text-align: center;
 }
+
+/**
+ * Centers the tooltip arrow to the tooltip horizontally.
+ */
+.slider :global(.tippy-arrow) {
```

Comment:
Had to add a global selector since we can't reference the shadow dom thumb, not ideal but seems stable, easy to catch if there are changes, easy to fix.